### PR TITLE
Fix not to break if Virtualmin plugin cannot be loaded

### PR DESCRIPTION
**Pull request by iliajie**

### CODE REVIEW

File: virtual-server-lib-funcs.pl [hunk 1]
```diff
@@ -19257,8 +19257,19 @@ sub load_plugin_libraries
...
+				&foreign_require($pname, "virtual_feature.pl");
+				};
+			if (!$@) {
+				$loaded++;
+				}
+			else {
+				my $err = $@;
+				$err = &html_strip($err);
+				$err =~ s/[\n\r]+/ /g;
+				&miniserv::log_error("Failed to load $pname : $err");
```
Comment (jcameron) [inline virtual-server-lib-funcs.pl:19271]: No no, you can't call miniserv functions from within webmin ... what if this code wasn't run via miniserv?

Instead, just print to STDERR

Comment (iliajie) [reply virtual-server-lib-funcs.pl]: Correct! Fixed on the latest commit.

File: virtual-server-lib-funcs.pl [hunk 2]
```diff
@@ -19257,8 +19257,24 @@ sub load_plugin_libraries
...
+				&foreign_require($pname, "virtual_feature.pl");
+				};
+			if (!$@) {
+				$loaded++;
+				}
+			else {
+				my $err = $@;
+				$err = &html_strip($err);
+				$err =~ s/[\n\r]+/ /g;
+				if (defined(&error_stderr)) {
```
Comment (jcameron) [inline virtual-server-lib-funcs.pl:19271]: Wait where this this declared? I've not seen this error_stderr function before..

Comment (iliajie) [reply virtual-server-lib-funcs.pl:19271]: It's new. I [just added](https://github.com/webmin/webmin/commit/8ef754b47754973944492444f981d9a913441957) it.

Comment (jcameron) [reply virtual-server-lib-funcs.pl:19271]: Ok makes sense!

**Pull request merged**